src/olympia/amo/monitors.py

```diff
@@ -185,3 +186,30 @@ def redis():
         status = ','.join(status)
 
     return status, redis_results
+
+
+def signer():
+    # Check Signing Server Endpoint
+    signer_results = None
+    status = ''
+
+    if getattr(settings, 'SIGNING_SERVER', False):
+        try:
+            response = requests.get('%s/status' % settings.SIGNING_SERVER,
+                                    timeout=settings.SIGNING_SERVER_TIMEOUT)
+            if response.status_code != 200:
+                status = 'Failed to chat with signing service.' \
+                         ' Invalid HTTP response code.'
```

Comment:
We usually do something like

```python
status = (
    'Failed to chat with signing service. '
    'Invalid HTTP response code.')
```

to wrap long strings to avoid the backslash escaping.